components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -662,10 +663,10 @@ export function LimitReachedModal(p: { children: React.ReactNode }) {
             </div>
             <div className="flex justify-end mt-6">
                 <a href={gitpodHostUrl.asDashboard().toString()}>
-                    <button className="secondary">Go to Dashboard</button>
+                    <Button variant="secondary">Go to Dashboard</Button>
                 </a>
                 <a href={gitpodHostUrl.with({ pathname: "plans" }).toString()} className="ml-2">
-                    <button>Upgrade</button>
+                    <Button>Upgrade</Button>
                 </a>
```

Comment:
We could invert both of these so that we have the anchors inside of the buttons, which will be `asChild` to remove duplicate focus targets. We can follow this up also

Reply:
I first went to make these just use `@podkit/buttons/LinkButton` - but our current implementation doesn't support an external or full url that has the protocol. I think we should fix the component to handle that, then we can have this just use LinkButton. [The suggested way](https://github.com/remix-run/react-router/issues/1147#issuecomment-113180884) is to render react-router's Link or just use an a tag if it's not a local, react-router aware link. I'll add an issue to the project to tackle this. It should be pretty simple but didn't want to blow this scope out more after looking for a few minutes.

Reply:
EXP-956 is the issue fyi